pkg/controller/remotemachineset/vsphereactuator.go

```diff
@@ -0,0 +1,74 @@
+package remotemachineset
+
+import (
+	"fmt"
+
+	machineapi "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
+	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
```

Comment:
Put "github.com/openshift/hive" imports in their own section at the end of the imports.